static/about_us.html

```diff
@@ -60,7 +60,7 @@
 				</div><!--Navbar header End-->
 				<nav class="collapse navbar-collapse navigation" id="bs-example-navbar-collapse-1" role="navigation">
 					<ul class="nav navbar-nav navbar-right ">
-						<li id="nav-bar-home"><a href="./index.html" class="">Home </a></li>
```

Comment:
The following lines are 3 times in the code. Please make a snippet out of this.
And don't forget to add a test.
I will not insist on correct localisation. Yet.